### DISCUSSION TITLE
Adding a Multiple Filter to a Grid

### DISCUSSION THREAD

**Discussion by Julian1993ARG** [serenity-is/Serenity#7183] [Help]
Hello, I need to add a multiple filter to a grid, but the data comes from an intermediate table. For example:
I have an employees table, a competencies table, and an employeeCompetencies table. I want to be able to select competencies in the employees grid and retrieve the employees who have those competencies. I want it to be a multiple filter.

**Comment by LSNicholls** [comment]
Look at the Users grid.  As you can see, there is a field that shows the Roles that a User belongs to, as a comma-delimited list.  You can also see that you can filter by Roles in the Users grid; you may not have realized that you can make multiple selections in that filter.  Try it and see!

I believe this behavior models exactly what you want to do, from a UX standpoint.  

From a data standpoint, I also believe it models your schema; in other words, there is a cross reference table (UserRoles) that is like your employeeCompetencies table, relating the Roles and Users tables.
